### ISSUE TITLE
When extension fails entire no other commands work

### ISSUE THREAD

**Issue by bmeverett** [laconalabs/LaconaApp#149]
I found this while developing an extension. If an extension for whatever reason isn't working no other command in Lacona works. This is fine for a developer who is developing an extension because they know it was their extension that is causing the problem. But, if something happens in production with an extension, then it shouldn't cause the entire program to fail as well.

**Comment by brandonhorst** [comment]
You're right - this is a problem with how the system is currently implemented. Right now, Errors that are thrown in synchronous sections of code (`describe` functions), and problems with parsing the returned `elliptical` elements should NOT cause errors. However, problems that occur in asynchronous bits of code will cause slowdowns across the whole system, and trying to load a command with syntax errors will break all commands.

This was improved with v1.1.0, but it still has a way to go.

**Comment by criztiano** [comment]
Same here: in my case the Applescript addon wasn't working (😥), and "blocked" off any other functionality. Deactivated, everything back to normal.
